main/midnight/Source/tme/scenario/item_character.cpp

```diff
@@ -1516,9 +1516,7 @@ namespace tme {
                         Cmd_PickupObject();
                         
                         // we might have dropped an object
-                        // and the pickup will have removed the object
-                        
-                        oinfo=NULL;
+                        // and the pickup will have removed the object                        
```

Comment:
Not sure if you are ok with fix. Issue here was that oinfo was being set to NULL only to be used on line 1538:
               if (seekMsg) {
                    mx->text->oinfo = oinfo ;
                    mx->SetLastActionMsg( mx->text->CookedSystemString( SS_SEEK, this) );
                }

CookedSystemString trying to use oinfo to create object description, that oinfohas been NULLED